tool/release/createnotes/Notes.kt

```diff
@@ -66,17 +51,21 @@ fun writeReleaseNoteMd(commitDescriptions: List<CommitDescription>, releaseTempl
 Install & Run: $installInstruction
 
 ## New Features
-${features.map(::createCommitNoteMd).joinToString("\n")}
+${features.map(::createNoteMd).joinToString("\n")}
 
 ## Bugs Fixed
-${bugs.map(::createCommitNoteMd).joinToString("\n")}
+${bugs.map(::createNoteMd).joinToString("\n")}
 
 ## Code Refactors
-${refactors.map(::createCommitNoteMd).joinToString("\n")}
+${refactors.map(::createNoteMd).joinToString("\n")}
 
 ## Other Improvements
-${others.map(::createCommitNoteMd).joinToString("\n")}
+${others.map(::createNoteMd).joinToString("\n")}
     """
 
     Files.write(releaseTemplateFile, template.replace(releaseTemplateRegex, markdown).toByteArray(UTF_8))
 }
+
+private fun createNoteMd(info: CommitInfo): String {
```

Comment:
@lolski this looks like it could be a method of `CommitInfo`? e.g. `CommitInfo::asMarkdown`

Reply:
I thought of that, but I want to concentrate the markdown generation in `Notes.kt`. `CommitInfo` should only contain relevant information about the commit, namely the title and the goal, without any knowledge about markdown.

Reply:
You know what, you made me rethink the whole thing and I think I arrive at a better level top-level structure!